in_toto/model.go

```diff
@@ -31,6 +33,24 @@ type Key struct {
 	Scheme              string   `json:"scheme"`
 }
 
+func validateKeyIdFormat(keyId string) error {
+	keyIdFormatCheck, err := regexp.MatchString("[a-fA-F0-9]+", keyId)
+	if err != nil {
+		return fmt.Errorf("unable to check if key ID has valid format")
+	}
```

Comment:
You can omit that `err != nil` check. It [only returns an error if it can't compile the pattern](https://golang.org/src/regexp/regexp.go?s=15411:15479#L481), which you control.